orc8r/cloud/go/obsidian/config.go

```diff
@@ -24,15 +24,21 @@ const (
 	DefaultStaticFolder  = "/var/opt/magma/static"
 	StaticURLPrefix      = "/apidocs"
 	ServiceName          = "OBSIDIAN"
+	// EnableDynamicSwaggerSpecs is a config in the obsidian
+	// service config.
+	// When true, poll and merge Swagger specs at runtime
```

Comment:
Nit: merge -> combine

I've been trying to differentiate between "combine", the high-level handling of logically combining multiple specs together, vs. "merge", the low-level implementation detail of how any two specs are merged